docs/querying/sql-data-types.md

```diff
@@ -140,23 +140,38 @@ as regular JSON arrays instead of in stringified form.
 
 ## NULL values
 
+By default, Druid treats NULL values closely to the SQL standard.
```

Comment:
I think we should be explicit and say we're talking about compatibility with ANSI SQL to avoid confusion and have ppl think we're talking about Druid SQL being compatible with Druid SQL

```suggestion
By default, Druid treats NULL values similarly to the ANSI SQL standard.
```